sys-block/arcconf/arcconf-2.01.22270.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_PV="v${PV//./_}"
+
+inherit linux-info
+
+DESCRIPTION="Microsemi Adaptec RAID Controller Command Line Utility"
+HOMEPAGE="http://www.adaptec.com/en-us/downloads/"
+SRC_URI="http://download.adaptec.com/raid/storage_manager/${PN}_${MY_PV}.zip"
+
+KEYWORDS="-* ~amd64 ~x86"
+LICENSE="Microsemi"
+SLOT="0"
+
+DEPEND="app-arch/unzip"
+RDEPEND="${RDEPEND}"
+
+RESTRICT="fetch mirror bindist"
+
+S="${WORKDIR}"
+
+pkg_nofetch() {
+	elog "Please download ${A} from"
+	elog "https://storage.microsemi.com/en-us/speed/raid/storage_manager/${PN}_${MY_PV}_zip.php"
+	elog "and place it in your DISTDIR directory."
+}
+
+pkg_setup() {
+	# CONFIG_HARDENED_USERCOPY_PAGESPAN makes ARCCONF segault
+	if linux-info_get_any_version && linux_config_src_exists ; then
+		CONFIG_CHECK="!HARDENED_USERCOPY_PAGESPAN"
+		check_extra_config
+	fi
+}
+
+src_install() {
+	exeinto /opt/arcconf
```

Comment:
Do you really need to create the whole directory for one file? Also if it's fully standalone, it's fine to install it to `/usr`.

Reply:
Okay, I've changed it to ```/usr/bin```. AFAIR, someone told me not to install binary only packages in ```/usr/bin```.

Reply:
That's a common misconception. `/opt` is meant for packages which don't follow FHS, and e.g. use Windows-style install layout (data located in special directories relative to executable. However, even some of those packages end up in `/usr/lib*/foo` (web browsers being the most common example).

Reply:
Thank you very much!